### ISSUE TITLE
No configuration setting found for key 'cacheDir'

### ISSUE THREAD

**Issue by lchamp87x** [clulab/eidos#584]
I get the following error

```
Exception in thread "main" com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'cacheDir'
	at com.typesafe.config.impl.SimpleConfig.findKeyOrNull(SimpleConfig.java:152)
	at com.typesafe.config.impl.SimpleConfig.findOrNull(SimpleConfig.java:170)
	at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:184)
	at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:189)
	at com.typesafe.config.impl.SimpleConfig.getString(SimpleConfig.java:246)
	at ai.lum.common.ConfigUtils$StringConfigFieldReader$.read(ConfigUtils.scala:88)
	at ai.lum.common.ConfigUtils$StringConfigFieldReader$.read(ConfigUtils.scala:86)
	at ai.lum.common.ConfigUtils$ConfigWrapper$.apply$extension(ConfigUtils.scala:36)
	at org.clulab.wm.eidos.apps.CacheOntologies$.delayedEndpoint$org$clulab$wm$eidos$apps$CacheOntologies$1(CacheOntologies.scala:16)
	at org.clulab.wm.eidos.apps.CacheOntologies$delayedInit$body.apply(CacheOntologies.scala:12)
	at scala.Function0.apply$mcV$sp(Function0.scala:34)
	at scala.Function0.apply$mcV$sp$(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App.$anonfun$main$1$adapted(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at scala.App.main(App.scala:76)
	at scala.App.main$(App.scala:74)
	at org.clulab.wm.eidos.apps.CacheOntologies$.main(CacheOntologies.scala:12)
	at org.clulab.wm.eidos.apps.CacheOntologies.main(CacheOntologies.scala)
```

when running 

`java -Xmx50g -cp target/scala-2.12/eidos-assembly-0.2.3-SNAPSHOT.jar org.clulab.wm.eidos.apps.CacheOntologies`

I am up to date from the master branch. And here is what my `eidos.conf` looks like,

```
EidosSystem {
// Override the default values here
             language = english
      masterRulesPath = /org/clulab/wm/eidos/${EidosSystem.language}/grammars/master.yml
         taxonomyPath = /org/clulab/wm/eidos/${EidosSystem.language}/grammars/taxonomy.yml
          hedgingPath  /org/clulab/wm/eidos/${EidosSystem.language}/confidence/hedging.txt
    timeNormModelPath = /org/clulab/wm/eidos/models/timenorm_model.hdf5
        entityFinders = ["gazetteer", "rulebased"]
          useTimeNorm = false
           useGeoNorm = false
 keepStatefulConcepts = true
}

filtering {
  stopWordsPath = /org/clulab/wm/eidos/${EidosSystem.language}/filtering/stops.txt
  transparentPath = /org/clulab/wm/eidos/${EidosSystem.language}/filtering/transparent.txt
}

actions {
  useCoref = true
  corefType = "causalBasic"
  taxonomyPath = ${EidosSystem.taxonomyPath}
  useExpansion = true
}

apps {
  inputDirectory = "."
  outputDirectory = "."
  inputFileExtension = ".txt"
  exportAs = ["serialized", "jsonld", "mitre"] // valid modes: jsonld, mitre, serialized
  groundTopN = 5
  ontologymapper {
    outfile = src/main/resources/org/clulab/wm/eidos/${EidosSystem.language}/ontologies/un_to_indicators.tsv
  }
}

ontologies {
  // W2V
  useW2V = true
  //  wordToVecPath = /org/clulab/wm/eidos/${EidosSystem.language}/w2v/vectors.txt
  wordToVecPath = /org/clulab/wm/eidos/${EidosSystem.language}/w2v/glove.840B.300d.txt

  // Activated Ontologies
  ontologies = ["un", "props", "mitre12", "who", "interventions"] // , "icasa", "mesh"]

  // Caching, for quick loading
  cacheDir = ./cache/${EidosSystem.language}
  useCache = false

  // Primary
  un            = /org/clulab/wm/eidos/${EidosSystem.language}/ontologies/un_ontology.yml
  props         = /org/clulab/wm/eidos/${EidosSystem.language}/ontologies/un_properties.yml
  // Plugins
  interventions = /org/clulab/wm/eidos/${EidosSystem.language}/ontologies/interventions.yml
  // Indicators
  mitre12       = /org/clulab/wm/eidos/${EidosSystem.language}/ontologies/mitre12_indicators.yml
  who           = /org/clulab/wm/eidos/${EidosSystem.language}/ontologies/who_ontology.yml
  // Variables
  icasa         = /org/clulab/wm/eidos/${EidosSystem.language}/ontologies/icasa.yml
  // Other
  mesh          = /org/clulab/wm/eidos/${EidosSystem.language}/ontologies/mesh_ontology.yml
}

geoparser {
  geoNormModelPath = /org/clulab/geonorm/model/geonorm_model.dl4j.zip
   geoWord2IdxPath = /org/clulab/wm/eidos/${EidosSystem.language}/context/word2idx_file.txt
     geoLoc2IdPath = /org/clulab/wm/eidos/${EidosSystem.language}/context/geo_dict_with_population_SOUTH_SUDAN.txt
}

adjectiveGrounder {
  domainParamKBPath = /org/clulab/wm/eidos/${EidosSystem.language}/quantifierKB/domain_parameters.kb
   quantifierKBPath = /org/clulab/wm/eidos/${EidosSystem.language}/quantifierKB/gradable_adj_fullmodel.kb
}
```

**Comment by kwalcock** [comment]
CacheOntologies doesn't create the directory for you and we didn't include one in the repo.  We'll probably change that, but in the meantime, just create a directory ./cache/english before running the program.

**Comment by kwalcock** [comment]
The statement above is incorrect.  Please stand by for a correction.

**Comment by kwalcock** [comment]
I think this should be fixed in master now with #587.
